src/UglyToad.PdfPig.Fonts/TrueType/TrueTypeFont.cs

```diff
@@ -11,6 +11,11 @@
     /// </summary>
     public class TrueTypeFont
     {
+        /// <summary>
+        /// FontFileBytes
+        /// </summary>
+        public IReadOnlyList<byte> FontFileBytes { get; }
```

Comment:
Presumably you need these bytes to pass to the corresponding graphics system, e.g. System.Drawing. I'd avoided adding them to prevent memory bloat in most client code scenarios. Could we instead store some way of accessing the bytes separately to the font object and only access them on demand, e.g. `PdfDocument.GetRawFontBytes(font)` or similar so we don't hold potentially large files in memory?

Reply:
Yes we need them to draw the font and yes it would be amazing to have `PdfDocument.GetRawFontBytes(font)`. I'm really not a fan of the what I did.